packages/bpk-component-star-rating/README.md

```diff
@@ -51,6 +51,7 @@ export default () => (
 | large          | bool                    | false    | false         |
 | maxRating      | number                  | false    | 5             |
 | rating         | number                  | false    | 0             |
+| roundRating    | bool                    | false    | false         |
```

Comment:
Can this be a function of type `number => number`

```suggestion
| roundRating    | func                    | false    | false         |
```

Reply:
It could be, the bigger problem I have right now is that something seems to think `roundRating` is a spelling error judging by the failed build 🤔

Reply:
If it's going to be a function instead of a bool I could call it `rounding` instead?

Reply:
Yeah that sounds like a better name 💯

Reply:
So I made the change, but looking over it again I feel like this goes against the spirit of the change we were trying to make, i.e. adding a secondary behaviour 🤔